state/upgrades_test.go

```diff
@@ -100,7 +100,7 @@ func (s *upgradesSuite) TestStripLocalUserDomainModels(c *gc.C) {
 	delete(initialModel, "txn-revno")
 	initialModel["owner"] = "test-admin"
 
-	// TODO (mattyw, cmars) We need to confirm that setting sla in this test
+	// TODO (mattyw, cmars) We need to confirm that setting sla and meter-status in this test
```

Comment:
What is the scenario this test attempts to recreate w.r.t SLAs? Upgrading models from a pre-SLA version of Juju?